example/android/build.gradle

```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```

Comment:
not sure why this changed. maybe flutter version that I ran flutter pub get ?

Reply:
I'm not sure, but this also ran fine locally and did not change on my machine, so seems fine.